acousticbrainz/data/__init__.py

```diff
@@ -139,13 +139,15 @@ def count_lowlevel(mbid):
 
 
 def get_summary_data(mbid):
+    """Fetches the lowlevel and highlevel features from abz database for the specified MBID."""
+    summary = {}
     mbid = str(mbid)
     conn = psycopg2.connect(current_app.config['PG_CONNECT'])
     cur = conn.cursor()
     try:
         cur.execute("SELECT data FROM lowlevel WHERE mbid = %s", (mbid, ))
         if not cur.rowcount:
-            raise NotFound("We do not have data for this track. Please upload it!")
+            raise NotFound
```

Comment:
You shouldn't use `NotFound` exception from werkzeug there. It's meant for use in case of HTTP errors, not our internal data fetching. So a better thing to do there would be to create your own exception.